src/stores/game.js

```diff
@@ -285,6 +287,12 @@ export const useGameStore = defineStore('game', {
     },
     opponentLeft() {
       this.players = this.players.filter((player) => player.pNum === this.myPNum);
+
+      if(this.myPNum === 0) {
+        this.p1Ready = false;
+      } else {
+        this.p0Ready = false;
+      }
```

Comment:
We already have `updateReady()` in the store, so I think we can utilize it like this

```suggestion

      updateReady(this.myPnum + 1 % 1)
```

Reply:
Okay Yea I understand you, maybe check if the opponent is ready, and then use updateReady()

Reply:
@Haviles04 I push the changes we agreed on.  I had to move it to the first line though, as the code that was already there deletes the opponent from the players array